shared/api/instance.go

```diff
@@ -49,6 +49,13 @@ type InstancesPost struct {
 	// Type (container or virtual-machine)
 	// Example: container
 	Type InstanceType `json:"type" yaml:"type"`
+
+	// Whether we want to rebuild the instance
+	Rebuild bool `json:"rebuild" yaml:"rebuild"`
```

Comment:
This doesn't tally up with https://discuss.linuxcontainers.org/t/adding-support-for-instance-rebuild/17195#api-changes-5

Which is right?